crates/core/src/rpc/utils.rs

```diff
@@ -206,18 +206,25 @@ pub fn transform_tx_metadata_to_ui_accounts(
     meta.inner_instructions
```

Comment:
Should this method take ownership of `meta` and transform the data, instead of cloning?
So
```rust
pub fn transform_tx_metadata_to_ui_accounts(
    meta: TransactionMetadata,
) 
```
and then use `into_iter()`
?